src/Symfony/Component/Messenger/Stamp/BusNameStamp.php

```diff
@@ -18,7 +18,7 @@
  *
  * @author Ryan Weaver <ryan@symfonycasts.com>
  */
-class BusNameStamp implements StampInterface
+final class BusNameStamp implements StampInterface
```

Comment:
I’m wondering if we should mark with @final and make final in 4.4 to minimize BC changes in patch releases.

Reply:
this PR is for 4.4, not a patch release